src/routes/login/+server.ts

```diff
@@ -0,0 +1,12 @@
+import { redirect } from "@sveltejs/kit";
+import { getOIDCAuthorizationUrl, getRedirectURI } from "$lib/server/auth";
+
+export async function GET({ url, locals }) {
+	// TODO: Handle errors if provider is not responding
+	const ssoAuthorizationUrl = await getOIDCAuthorizationUrl(
+		{ redirectURI: getRedirectURI(url) },
+		{ sessionId: locals.sessionId }
+	);
+
+	throw redirect(303, ssoAuthorizationUrl);
+}
```

Comment:
Apologies :pray: 

We should go back to a POST, just to make sure the user clicked the "sign in" button and so we can set the `"ethicsModalAcceptedAt"` with peace of mind. (i.e. they didn't follow a link to hf.co/chat/login from someone else without reading the modal)